modules/card.js

```diff
@@ -6,6 +6,7 @@ class MtgCardLoader {
     constructor() {
         this.cardApi = "https://api.magicthegathering.io/v1/cards?name=";
         this.commands = ["card"];
+        this.maxLength = 2000;
```

Comment:
Question is how we determine those 10. I want to tackle better search at some point, but I can't promise when I get around to doing that.